agent/src/beerocks/slave/backhaul_manager/backhaul_manager_thread.cpp

```diff
@@ -1781,11 +1781,8 @@ bool backhaul_manager::handle_1905_topology_query(ieee1905_1::CmduMessageRx &cmd
                    << (int)mid;
         return false;
     }
-    tlvDeviceInformation->mac() = network_utils::mac_from_string(bridge_info.mac);
 
-    // https://github.com/prplfoundation/prplMesh/issues/300
-    //TODO: set number of local interfaces.
-    //TODO: fill info of each of the local interfaces, according to IEEE_1905 section 6.4.5
+    tlvDeviceInformation->mac() = network_utils::mac_from_string(bridge_info.mac);
```

Comment:
There's something wrong here... The commit message title is "add DeviceInformation TLV", but the TLV was already there, and the only thing that happens is that the TODOs are removed.

I assume that this commit should be squashed with the last commit - but even then, the TODOs should probably not be removed, just rewritten.

Reply:
Done